wandb/cli/cli.py

```diff
@@ -2864,30 +2864,3 @@ def verify(host):
         and url_success
     ):
         sys.exit(1)
-
```

Comment:
just curious why did you remove the cli support?

Reply:
We can add it back in a future update.  I didn't want to clog up this release with something that is not well tested

Reply:
Atm, we mainly call the importer through python anyways

Reply:
got it! thanks!